Show columns clarifications (#1826)
neo4j/docs-operations: modules/ROOT/pages/authentication-authorization/manage-privileges.adoc

When using the `RETURN` clause, the `YIELD` clause is mandatory and must not be omitted.

For an easy overview of the existing privileges, it is recommended to use the `AS COMMANDS` version of the `SHOW` command.
This returns two columns:

> [options="header", width="100%", cols="2a,4,2m"]

Reviewer: ```suggestion
.`SHOW PRIVILEGES AS COMMANDS` output
[options="header", width="100%", cols="2a,4,2m"]
```

|===
| Column
| Description
| Type

| command
| The privilege as the command that is granted or denied.
Or in the `AS REVOKE COMMANDS` case, the command to revoke the privilege. label:default-output[]
| STRING

| immutable
| Whether or not the privilege is immutable.
| BOOLEAN
|===

> When omitting the `AS COMMANDS` clause, results will include multiple columns describing privileges instead.

Reviewer: ```suggestion
Alternatively, you can omit the `AS COMMANDS` clause and get the full details of the privileges returned in multiple columns:
```

> They are all returned by default.

Reviewer: This sounds like some of the details are not returned by default. I would suggest to remove it.
```suggestion
```

Author: Not sure what you mean? they are all default columns returned without a YIELD which is what this is marking. A lot of other tables like this has either another column (or two) showing if they are returned by default for the command or need a yield or they have the `default` label on them. But since that would be true for all rows in this table we just have this comment instead.

If this is about the column only returned by the `SHOW USER PRIVILEGE` versions, then it is still a default column by the definition we use it for (not requiring YIELD) it is just not available for all versions of the command.

Reviewer: I couldn't get from the message that the default was without YIELD.

Author: I think it comes from that being the language we have used throughout the admin command sections, we talk about what is returned by default and what columns you need a YIELD for 🤷

> [options="header", width="100%", cols="4m,6a,2m"]

Reviewer: ```suggestion
.`SHOW PRIVILEGES` output
[options="header", width="100%", cols="4m,6a,2m"]
```
